frontend/src/liboperator/api/v1/messages.go

```diff
@@ -227,3 +227,7 @@ type EmptyResponse struct{}
 
 type StopCVDResponse = EmptyResponse
 
+type UUID struct {
```

Comment:
A full struct to represent a uuid is overkill, just a string would be enough.

Reply:
True, was under the impression a single string was not a valid JSON value.